workshop/content/exercises/deploy-sample-app.adoc

```diff
@@ -10,20 +10,28 @@ We can verify the name of the project with:
 oc project -q
 ----
 
-You will use the link:https://github.com/sclorg/nodejs-ex[nodejs-ex] sample application during this tutorial, which is a simple nodejs application.
+You will use the link:https://github.com/sclorg/nodejs-ex[nodejs-ex] sample application during this tutorial,
+which is a simple Node.js application.
 
-Create the Kubernetes objects for deploying the nodejs-ex app on OpenShift by running
-the command below. The deployment will not complete since there are no container images
-built for the nodejs-ex application yet. That you will do in the following sections
-through a CI/CD pipeline.
+To prepare for nodejs-ex's eventual deployment, you will create Kubernetes objects that
```

Comment:
Added this text earlier in this section to highlight that resources being created do not actually deploy an application but will be used to support the application that is eventually to be deployed.

Reply:
Good changes here, I dig it.